src/node/node_function.hpp

```diff
@@ -38,14 +38,6 @@ inline v8::Local<v8::Value> node::Function::call(v8::Isolate* isolate, const v8:
 
 template<>
 inline v8::Local<v8::Value> node::Function::callback(v8::Isolate* isolate, const v8::Local<v8::Function> &function, const v8::Local<v8::Object> &this_object, size_t argc, const v8::Local<v8::Value> arguments[]) {
-    if (!isolate->GetCallingContext().IsEmpty()) {
```

Comment:
v8 has removed `GetCallingContext()` because of tail calls. I don't know how to implement this optimization otherwise.